### ISSUE TITLE
Lootr advancements shows “lootr.advancements.”

### ISSUE THREAD

**Issue by SokaMP4** [noobanidus/Lootr#328]
**ver**: _lootr-forge-1.18.2-0.3.28.70.jar_

**Issue**: advancedments Tab of Lootr shows “lootr.advancements.”, it does not show the
proper title and description.

**add. info.**: I use a modpack and I suspect one
of my mods either conflicting it or overriding, 
although it seems to be too unrelated for it to be
the advancements tab

**Comment by noobanidus** [comment]
This is a bit of a strange one as the translations work fine in my development environment and the fix I made previously should've resolved this. 

Are you able to test in a single-player new world with the `/lootr chest` command and then attempting to break blocks to see if it is all translations or just the advancements?

**Comment by noobanidus** [comment]
I've just tried a new mod pack with only Lootr installed directly from CurseForge for 1.18.2, and the advancements are showing up fine for me.

Can you confirm that there are no other versions of Lootr in your mods folder that may be loaded before this? A previous version of Lootr was problematic.

Failing that, can you try removing then reinstalling Lootr?

If that still does not work, I'm not sure what the issue could be.

**Comment by SokaMP4** [comment]
Hello! I have tried what you have mentioned and this is what I got:
(these were from the chat)
After Entering Command: lootr.commands.create 
After Breaking Chest: lootr.message.should_sneak
                                 lootr.message.should_sneak2
                                 
I have tried reinstalling and it worked! I saw that I have installed a older version of lootr.

Anyhow, thank you so much for your time it is very much appreciated
